File: fontes/c/micro/micro04.c
#include<stdio.h>

int main()
{
 int x;
 int num;
 int intervalo = 0;

 for(x = 0; x < 5; x++)
 {
 	printf("Digite um número: ");
 	scanf("%d", &num);
	if (num >= 10)
	{
	 	if (num <= 150)
	 	{
	 		intervalo = intervalo + 1;
	 	}
	}
 }
 printf("Ao total, foram digitados %d números no intervalo entre 10 e 150\n", intervalo);
 return 0;
}

File: fontes/pascal/micro/micro04.pas
program Micro04;
    
var 
	x, num, intervalo: INTEGER;
begin

    for x:= 1 to 5 do 
        begin
            writeln('Escreva um numero: ');
            readln(num);
            if num>=10 then
                if num<=150 then
                    intervalo:=intervalo+1;
        end;
    
    writeln('Ao total, foram digitados ',intervalo,' números no intervalo entre 10 e 150')
end.
